packages-ts/gauntlet-terra/src/commands/middlewares.ts

```diff
@@ -37,7 +37,7 @@ export const withWallet: Middleware = async (c: TerraCommand, next: Next) => {
   }
 
   c.wallet = c.provider.wallet(key)
-  logger.info(`Operator address is ${c.wallet.key.accAddress}`)
+  logger.debug(`Operator address is ${c.wallet.key.accAddress}`)
```

Comment:
Multisig is currently wrapping 3 commands, and this can become noisy, as will be prompted 3 times in that case. I guess we could move the log somewhere else